src/lib.rs

```diff
@@ -736,8 +736,8 @@ impl Builder {
     }
 
     /// Avoid generating any unstable Rust, such as Rust unions, in the generated bindings.
-    pub fn no_unstable_rust(mut self) -> Builder {
-        self.options.unstable_rust = false;
+    pub fn unstable_rust(mut self) -> Builder {
```

Comment:
While we're breaking the API, should we take a boolean just for consistency?

i.e., `fn unstable_rust(mut self, doit: bool) -> Self`?